stories/topics/con-aws-application-architecture.adoc

```diff
@@ -8,6 +8,11 @@
 
 image::aap-on-aws-architecture.png[AWS application architecture]
 
+== Architecture Description
+
+{PlatformName} from {Market} was architected to be private, with no public access or public egress installed by default. This requires customers to expose their infrastructure manually. Components deployed are isolated into their own VPC. Applications run on instances backed by an autoscale group, which run on RHEL-based AMIs as their basis. These AMIs are preloaded with all the required container images to run the {PlatformNameShort} hub, controller, and receptor components. A shared EFS volume is mounted into each instance stood up by the autoscale groups.
```

Comment:
was designed to be private
hub =(HubName}
controller ={ControllerName}
receptor ={ReceptorRPM}

Reply:
Updated!